apps/admin-gui/src/app/shared/components/dialogs/edit-application-form-item-dialog/edit-application-form-item-dialog.component.ts

```diff
@@ -123,35 +117,41 @@ export class EditApplicationFormItemDialogComponent implements OnInit {
 
   getDestinationAndSourceAttributes() {
     this.translateService.get('DIALOGS.APPLICATION_FORM_EDIT_ITEM.NO_SELECTED_ITEM').subscribe( noItem => {
-      this.sourceAttributes.push(new SelectionItem(noItem, ''));
-      this.destinationAttributes.push(new SelectionItem(noItem, ''));
+      const emptyItem: SelectionItem = new SelectionItem(noItem, '');
+      this.sourceAttributes.push(emptyItem);
+      this.destinationAttributes.push(emptyItem);
+      this.sourceAtt = emptyItem;
+      this.destinationAtt = emptyItem;
     });
 
     for (const attribute of this.attributeDefinitions) {
+      const item: SelectionItem = new SelectionItem(attribute.friendlyName + ' (' + attribute.entity + ' / ' + this.getDefinition(attribute) + ')',
+        attribute.namespace + ':' + attribute.friendlyName);
+      if(item.value === this.applicationFormItem.perunSourceAttribute){
+        this.sourceAtt = item;
+      }
+      if(item.value === this.applicationFormItem.perunDestinationAttribute){
+        this.destinationAtt = item;
+      }
+
       if (attribute.entity.toLowerCase() === 'user' || attribute.entity.toLowerCase() === 'member') {
         // add only member and user attributes
-        this.sourceAttributes.push(
-          new SelectionItem(attribute.friendlyName + ' (' + attribute.entity + ' / ' + this.getDefinition(attribute) + ')',
-            attribute.namespace + ':' + attribute.friendlyName)
-        );
-        this.destinationAttributes.push(
-          new SelectionItem(attribute.friendlyName + ' (' + attribute.entity + ' / ' + this.getDefinition(attribute) + ')',
-            attribute.namespace + ':' + attribute.friendlyName)
-        );
+        this.sourceAttributes.push(item);
+        this.destinationAttributes.push(item);
       } else if (attribute.entity.toLowerCase() === 'vo') {
         // source attributes can be VO too
-        this.sourceAttributes.push(
-          new SelectionItem(attribute.friendlyName + ' (' + attribute.entity + ' / ' + this.getDefinition(attribute) + ')',
-            attribute.namespace + ':' + attribute.friendlyName)
-        );
+        this.sourceAttributes.push(item);
       } else if (attribute.entity.toLowerCase() === 'group' && this.data.group) {
         // if dialog is for group
-        this.sourceAttributes.push(
-          new SelectionItem(attribute.friendlyName + ' (' + attribute.entity + ' / ' + this.getDefinition(attribute) + ')',
-            attribute.namespace + ':' + attribute.friendlyName)
-        );
+        this.sourceAttributes.push(item);
       }
     }
+    this.sortAttributes(this.sourceAttributes);
+    this.sortAttributes(this.destinationAttributes);
+  }
+
+  sortAttributes(attributes: SelectionItem[]) {
+    attributes.sort((a: SelectionItem, b: SelectionItem) => a.displayName.localeCompare(b.displayName));
```

Comment:
Sorting items is nice but I would expect that value `No item selected` would be always on top of the list